include/audit_events.hpp

```diff
@@ -242,7 +239,17 @@ inline void auditEvent(const crow::Request& req, const std::string& userName,
     }
 
     // encode user account name to ensure it is in an appropriate format
-    user = audit_encode_nv_string("acct", userName.c_str(), 0);
+    size_t userLen = 0;
+    char* user = audit_encode_nv_string("acct", userName.c_str(), 0);
+
+    // setup a unique_ptr to handle freeing memory from user
+    std::unique_ptr<char, void (*)(char*)> userUP(user, [](char* ptr) {
+        if (ptr != nullptr)
+        {
+            ::free(ptr);
+        }
+    });
+
     if (user == NULL)
     {
         BMCWEB_LOG_ERROR << "Error appending user to audit msg : " << errno;
```

Comment:
BMCWEB_LOG_ERROR  (in L255 ) seems not a fatal error case, but more of informational or warning message?

Reply:
Agreed, I changed to warning.